katas/args/alejandra/args.test.js

```diff
@@ -0,0 +1,20 @@
+import { splitCommand } from './argsParser.js';
```

Comment:
This file is causing some issues since it has the same name as `Args.test.js`, just that first "a" is lowercase. I think we could remove this one? or if needed, please rename so it does not conflict with the other file. You can use `git mv` command